### ISSUE TITLE
Test gradient of Direct composed with Forward

### ISSUE THREAD

**Issue by lostanlen** [kymatio/murenn#19]
Example

```python
N = 2**10
x = torch.zeros(1, 1, N, requires_grad=True)
J = 8

dtcwt = murenn.DTCWT(J=J)
idtcwt = murenn.DTCWTInverse(J=J)

x_phi, x_psis = dtcwt(x)
y = idtcwt((x_phi, x_psis))
y0 = y[0, 0, N//2]
y0.backward()
```

Check that `x.grad` is a Dirac ☺️

**Comment by lostanlen** [comment]
It would be good to make this test multichannel so as to preemptively catch bugs like #20 (now fixed) ☺️

**Comment by lostanlen** [comment]
also let's make sure to test the case `normalize=False`. It looks like it is not differentiable right now, because of some in-place operation

```python
if (j%2 == 1) and self.alternate_gh:
    # The result is anti-analytic in the Hilbert sense.
    # We conjugate the result to bring the spectrum back to (0, pi).
    # This is purely by convention and for consistency through j.
    x_psi_i *= -1
```
try `x_psi_i = x_psi_i * (-1)`

**Comment by xir4n** [comment]
> It would be good to make this test multichannel so as to preemptively catch bugs like #20 (now fixed) ☺️

Thanks! You are maximally right. 
But since `x.backward()` requires `x` to be a scalar, making this test multichannel is difficult for me. What I've done is to test the gradients in multichannel case by using `gradcheck()` for each torch `torch.autograd.Function` instance.

**Comment by xir4n** [comment]
> also let's make sure to test the case `normalize=False`. It looks like it is not differentiable right now, because of some in-place operation
> 
> ```python
> if (j%2 == 1) and self.alternate_gh:
>     # The result is anti-analytic in the Hilbert sense.
>     # We conjugate the result to bring the spectrum back to (0, pi).
>     # This is purely by convention and for consistency through j.
>     x_psi_i *= -1
> ```
> 
> try `x_psi_i = x_psi_i * (-1)`

Yes, you are right, thank you!

**Comment by lostanlen** [comment]
I'll write the test myself then.

**Comment by xir4n** [comment]
Okay, thanks. I'm sorry about that.
